### DISCUSSION TITLE
Should Select component have hover-effects, such as hand pointer icon?

### DISCUSSION THREAD

**Discussion by jawsmartin** [Shopify/polaris-react#8898] [Ideas] [answered]
Should the [Select component](https://polaris.shopify.com/components/selection-and-input/select) have a hover effect when interacting with it? See the [Action list component](https://polaris.shopify.com/components/lists/action-list). I would expect the cursor to update to the hand pointer icon and the background color to change. 

https://user-images.githubusercontent.com/3891708/230956964-7deea987-6979-4d59-b565-8b3c75338328.mov

**Comment by yurm04** [comment, marked as answer]
I'm not sure if this was a bug or an intentional design choice at the time. But, with v12 our `Select` component does have a hover state

https://github.com/Shopify/polaris/assets/4642404/dec5ba5a-9ecb-4de1-9449-edd8a55d9620
